src/handlers/tracker.ts

```diff
@@ -58,14 +58,19 @@ export class TrackerHandler implements IDisposable {
     this.consoleTracker = options.consoleTracker;
     this.editorTracker = options.editorTracker;
 
-    this.readOnlyEditorFactory = new ReadOnlyEditorFactory({
-      editorServices: options.editorServices
-    });
-
-    this.readOnlyEditorTracker = new WidgetTracker<
-      MainAreaWidget<CodeEditorWrapper>
-    >({
-      namespace: '@jupyterlab/debugger'
+    this.debuggerService.modelChanged.connect(() => {
+      const debuggerModel = this.debuggerService.model as Debugger.Model;
+      if (debuggerModel) {
+        debuggerModel.callstack.currentFrameChanged.connect(
+          this.onCurrentFrameChanged,
+          this
+        );
+
+        debuggerModel.breakpoints.clicked.connect((_, breakpoint) => {
+          const debugSessionPath = this.debuggerService.session.client.path;
+          this.find(debugSessionPath, breakpoint.source.path);
+        });
+      }
```

Comment:
This snippet of code is already in the `onModelChanged` below. Is there a reason to have it here too?

Reply:
Yeah, it looks like the lines below this chunk need to be removed?